packages/itwinui-css/src/tabs/base.scss

```diff
@@ -138,31 +142,28 @@ $borderless-horizontal-tab-min-height: calc(
       min-block-size: $borderless-horizontal-tab-min-height;
     }
 
-    &[data-iui-overflow='true'] {
```

Comment:
We want to remove this data attribute but it's used in setting white-space nowrap in overflow scenarios.
<img width="315" alt="image" src="https://github.com/iTwin/iTwinUI/assets/83585998/e3476dc7-cf99-4c6f-8d3b-f2fd12d3c89e">

We are using wrapping with long labels and descriptions:
<img width="1110" alt="image" src="https://github.com/iTwin/iTwinUI/assets/83585998/7da788a0-d44b-4d10-a13f-0cf00a61795b">

Is it better to remove this nowrap and have tabs like this when overflowing: 
<img width="297" alt="image" src="https://github.com/iTwin/iTwinUI/assets/83585998/77feb98d-1006-4151-9065-15198974698d">

Or add it to tabs in general and have long labels not wrapping:
<img width="967" alt="image" src="https://github.com/iTwin/iTwinUI/assets/83585998/1660c70e-b10f-4cf9-a6d8-f76492eec5b5">

Or keep data attribute?

Reply:
I usually prefer the unwrapped version with something like a [`title`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#result_2)/tooltip upon hovering. Maybe we keep the unwrapped as the default and offer a prop to customize this?

Reply:
Decided to remove this nowrap from tabs.

Reply:
i think we can use `flex-shrink: 0` on individual tabs to prevent them from wrapping when overflowing.

Reply:
Works perfectly for overflow, long labels does get big. Are we ok with this? 
![image](https://github.com/iTwin/iTwinUI/assets/83585998/32b49a86-17ac-4806-a669-b01b648f65d3)

Reply:
Updated with https://github.com/iTwin/iTwinUI/pull/1548/commits/6dbb70fed6756dc73569f7629dd949246974c9cc